README.md

```diff
@@ -9,7 +9,7 @@ Basic:
 steps:
 - uses: huaxk/postgis-action@v1
   with:
-    postgresql version: '11'  # See https://https://hub.docker.com/r/mdillon/postgis for available versions, if it is not specified, use the default value 'latest'
+    postgresql version: '11'  # See https://hub.docker.com/r/mdillon/postgis for available versions, if it is not specified, use the default value 'latest'
```

Comment:
This action actually uses `postgis/postgis`:

https://github.com/huaxk/postgis-action/blob/0e3a7a41c0b06b9a1fe690d937f21168af8839bd/entrypoint.sh#L7

So I adapted the URL here, as well as the tag in the example to match one that is actually available.

```suggestion
    postgresql version: '11-3.2-alpine'  # See https://hub.docker.com/r/postgis/postgis for available tags, if it is not specified, use the default value `latest`
```